api/test/robot_tests/keywords.resource

```diff
@@ -60,7 +60,7 @@ Get SC address
     [Documentation]    Get the address of the SC from a string containing the address after ':' (colon)
     [Arguments]    ${string}
     ${words}=    Split String    ${string}    :
-    ${sc_address}=    Get From List    ${words}    1
+    ${sc_address}=    Get From List    ${words}    0
```

Comment:
idk i was having an index out of bounds error, practically it works the same, just instead of looking at index 1 we look at index 0 where the data is

Reply:
I understand the issue now
Before, `api/test/robot_tests/cmd/cmd.robot` line 40, we were using "data" which is a string containing some text followed by ":AS...". That's why we needed this function. But as you changed it for "address", you directly get the address so you don't need any parsing. So you can simply remove the call to this function in tests you modified and that are using "address" instead of "data".

Reply:
okay ty!!

Reply:
```
.json response: {'firstEvent': {'address': 'AS12ngeswNNDYz9VUWzCfnigLWkuGyCtt5a3x7QYM5M1kriAqVkPu', 'data': 'TestSC Constructor called'}, 'operationId': 'O12pKG41Hi7V3y65DFH2g9KrAEfL4jFL8p43mQxdg8Vrp4BXB5ac'}
POST a Smart Contract                                                 | FAIL |
Keyword name cannot be empty.
```

i think i still need the parsing tho :/

Reply:
which would make sense because i haven't modified the response